example/src/App.tsx

```diff
@@ -210,10 +224,18 @@ const AppContainer = () => {
   const [shouldScroll, setShouldScroll] = useState<boolean>(true);
   const [currentPlaying, setCurrentPlaying] = useState<string>('');
   const [list, setList] = useState<ListItem[]>(audioListArray);
+  const [currentPlaybackSpeed, setCurrentPlaybackSpeed] =
+    useState<PlaybackSpeedType>(1.0);
```

Comment:
Make sure if user add values outside of our range then set value to default in our side